src/zos_jobs/zowe/zos_jobs_for_zowe_sdk/jobs.py

```diff
@@ -117,6 +117,38 @@ def delete_job(self, jobname, jobid, modify_version="2.0"):
         response_json = self.request_handler.perform_request("DELETE", custom_args, expected_code=[202, 200])
         return response_json
 
+    def change_jobs_class(self, jobname: str, jobid: str, class_name: str, modify_version="2.0"):
+        """Changes the job class
+
+        Parameters
+        ----------
+        jobname: str
+            The name of the job
+        jobid: str
+            The job id on JES
+        modify_version: str
+            Default ("2.0") specifies that the request is to be processed synchronously. For asynchronous processing - change the value to "1.0"
+
+        Returns
+        -------
+        response_json
+            A JSON containing the result of the request execution
+        """
+        if modify_version not in ("1.0", "2.0"):
+            raise ValueError('Accepted values for modify_version: "1.0" or "2.0"')
```

Comment:
Could you please add a unit test verifying that this error is thrown for an invalid value of `modify_version`?